doc/genstdlib.jl

```diff
@@ -0,0 +1,80 @@
+using .Markdown
```

Comment:
This file is missing a header, as well as using 2 space indentation (rather than 4).